kanidmd/daemon/src/main.rs

```diff
@@ -293,45 +293,60 @@ async fn main() {
                         }
                     }
 
+                    #[cfg(debug_assertions)]
+                    trace!("Starting server core...");
```

Comment:
Rather than these traces could you do a #instrument around create_server_core instead at level trace? No need for the debug assert.

Reply:
Example https://github.com/kanidm/kanidm/blob/master/kanidmd/core/src/actors/v1_read.rs#L439 change level to trace, and omit the "fields" line.

Reply:
oh this was me trying to find the stack dump... I'll remove them.